EventFilter/GEMRawToDigi/plugins/GEMDigiToRawModule.cc

```diff
@@ -40,18 +40,24 @@ class GEMDigiToRawModule : public edm::global::EDProducer<edm::RunCache<GEMChMap
 
 private:
   int event_type_;
+  int minBunch_;
+  int maxBunch_;
   edm::EDGetTokenT<GEMDigiCollection> digi_token;
```

Comment:
```suggestion
  const edm::EDGetTokenT<GEMDigiCollection> digiToken_;
```
(besides making it `const` , you should use the same naming scheme used elsewhere, i.e. trailing underscore for class members, "Token" instead of "_token")